fiaas_skipper/__init__.py

```diff
@@ -74,18 +76,24 @@ def main():
                 log.exception("Unable to load spec config extension file {!r} using defaults"
                               .format(cfg.spec_file_override))
         if cfg.enable_crd_support:
+            application = FiaasApplication
             deployer = CrdDeployer(cluster=cluster, release_channel_factory=release_channel_factory,
                                    bootstrap=CrdBootstrapper(), spec_config_extension=spec_config_extension)
         elif cfg.enable_tpr_support:
+            application = PaasbetaApplication
             deployer = TprDeployer(cluster=cluster, release_channel_factory=release_channel_factory,
                                    bootstrap=TprBootstrapper(), spec_config_extension=spec_config_extension)
+        # Do period checking of deployment status across all namespaces
+        status_tracker = StatusTracker(cluster=cluster, application=application)
+        status_tracker.start()
         if not cfg.disable_autoupdate:
-            updater = AutoUpdater(release_channel_factory=release_channel_factory, deployer=deployer)
+            updater = AutoUpdater(release_channel_factory=release_channel_factory, deployer=deployer,
+                                  status=status_tracker)
             updater.daemon = True
```

Comment:
I know it's not from this PR, but the same thing about the daemon flag makes sense here as well.

Reply:
I'll fix that as well.